gulpfile.js

```diff
@@ -2,7 +2,8 @@ var gulp = require('gulp'),
     minimist = require('minimist'),
     summary = require('jshint-summary'),
     del = require('del'),
-    plugins = require('gulp-load-plugins')();
+    plugins = require('gulp-load-plugins')(),
+    connect = require('gulp-connect');
```

Comment:
It's starting by `gulp-`, you don't have to require it, it's already managed by `gulp-load-plugins`. To use it, a `plugins.connect` is enough.